src/audiences/edit/components/select-include.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const { __ } = wp.i18n;
+
+const StyledSelect = styled.select`
+	vertical-align: middle;
+
+	&:not(:hover, :focus) {
+		border: none;
+		background: none;
+		margin-right: 8px;
+	}
+`;
+
+const SelectInclude = ( { value, onChange, label = '', name } ) => {
+	const id = name.replace( /\W+/g, '-' ).replace( /^\W+.*?\W+$/, '' );
+	return (
+		<StyledSelect
+			id={ id }
+			onChange={ onChange }
+			value={ value }
+			name={ name }
```

Comment:
Could maybe do something like `const SelectInclude = ( { label = '', ...selectProps } )` then pass them here instead of unpacking and repacking?